ElementX/Sources/UITests/UITestsAppCoordinator.swift

```diff
@@ -275,10 +275,29 @@ class MockScreen: Identifiable {
         case .roomDetailsScreen:
             let navigationStackCoordinator = NavigationStackCoordinator()
             let userNotificationController = UserNotificationController(rootCoordinator: navigationStackCoordinator)
+            var roomProxy = MockRoomProxy(
+                displayName: "Room",
+                isEncrypted: true,
+                members: [.mockAlice, .mockBob, .mockCharlie]
+            )
+            roomProxy.id = "MockRoomIdentifier"
```

Comment:
Style and you can use the synthesised init for the `id`.
```suggestion
            var roomProxy = MockRoomProxy(id: "MockRoomIdentifier",
                                          displayName: "Room",
                                          isEncrypted: true,
                                          members: [.mockAlice, .mockBob, .mockCharlie])
```

Reply:
done